src/test/java/utils/DriverSetUp.java

```diff
@@ -0,0 +1,41 @@
+package utils;
+
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.*;
+
+import java.io.*;
+import java.util.Properties;
+
+public class DriverSetUp {
+    protected static WebDriver driver;
+    private String filePath = "/Users/avoloshchuk/IdeaProjects/com.sharelane/src/test/resources/app.properties";
+    private Properties appProps = new Properties();
+
+    public void loadPropertiesFromFile() {
+        try (InputStream input = new FileInputStream(filePath)) {
+            appProps.load(input);
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
please use any general logger and push out not the hole stack trace , but any descriptive and short error

Reply:
e.printStackTrace() was changed to e.getMessage()

Reply:
still don't see the change you've mentioned.... 
BRW, I guess I wasn't clear - by 'general logging of err msg' I meant smth like
` logger.error("Some err msg: {}", e.getMessage());`